Sofa/Component/Mass/tests/DiagonalMass_test.cpp

```diff
@@ -101,6 +101,7 @@ class DiagonalMass_test : public BaseTest
         sofa::simpleapi::importPlugin("Sofa.Component.Topology.Container.Dynamic");
         sofa::simpleapi::importPlugin("Sofa.Component.Topology.Container.Grid");
         sofa::simpleapi::importPlugin("Sofa.Component.StateContainer");
+        sofa::simpleapi::importPlugin("Sofa.Component.Mass");
```

Comment:
Unrelated to the PR: I find it confusing that `simpleapi::importPlugin` does something more than `pluginManager.loadPlugin`, and it is not obvious which one should be preferred.